airbyte-integrations/connectors/source-oracle-v2/src/cdk/test/kotlin/io/airbyte/cdk/jdbc/H2TestFixtureTest.kt

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (c) 2024 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.cdk.jdbc
+
+import java.sql.DriverManager
+import org.junit.jupiter.api.Assertions
+import org.junit.jupiter.api.Test
+
+class H2TestFixtureTest {
+
+    val h2 = H2TestFixture()
```

Comment:
... what's H2?

Reply:
It's java's equivalent of sqlite: https://en.wikipedia.org/wiki/H2_(database)